### PR TITLE
Add docs around RDS Postgres extension functions

### PR DESCRIPTION
**Pull request by whummer**
Add docs around RDS Postgres extension functions. This information has recently been requested by a user in the Community Slack channel. /cc @komarkovich

### CODE REVIEW

File: content/en/user-guide/aws/rds/index.md [hunk 1]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
...
+
+LocalStack supports certain extensions and functions that are provided in RDS to interact with other AWS services.
+At the moment, primarily extension functions for the PostgreSQL engine are supported.
+
+### Using the `aws_lambda` extension to interact with AWS Lambda
+
+The [`aws_lambda` extension](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL-Lambda.html) can be used in local RDS PostgreSQL databases to interact with the Lambda API.
+
+For example, in the SQL code snippet below, we are loading the `aws_lambda` extension, then generate a full ARN from a function name, and finally invoke the Lambda function directly from the SQL query:
+```
```
Comment (bentsku) [inline content/en/user-guide/aws/rds/index.md:283]:
```suggestion
```sql
```

File: content/en/user-guide/aws/rds/index.md [hunk 2]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
...
+SELECT aws_lambda.invoke('my_function', '{\"body\": \"Hello!\"}'::json);
+```
+
+### Using the `aws_s3` extension to interact with AWS S3
+
+The [`aws_s3` extension](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/postgresql-s3-export.html) can be used in local RDS PostgreSQL databases to interact with the S3 API.
+
+In the SQL code snippet below, we are loading the `aws_s3` extension, then use the `table_import_from_s3(..)` function to populate the data in a table `table1` from a CSV file `test.csv` stored in a local S3 bucket `mybucket1`:
+
+```
```
Comment (bentsku) [inline content/en/user-guide/aws/rds/index.md:297]:
```suggestion
```sql
```

File: content/en/user-guide/aws/rds/index.md [hunk 3]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
...
+CREATE EXTENSION IF NOT EXISTS aws_s3 CASCADE;
+SELECT aws_s3.table_import_from_s3(
+    'table1', 'c1, c2, c3', '(format csv)',
+    aws_commons.create_s3_uri('mybucket1', 'test.csv', 'us-east-1')
+)
+```
+
+Analogously, we can use the `query_export_to_s3(..)` extension function to export data from a table `table2` into a CSV file `test.csv` in local S3 bucket `mybucket2`:
+```
```
Comment (bentsku) [inline content/en/user-guide/aws/rds/index.md:306]:
```suggestion
```sql
```

**Review by bentsku — APPROVED** [review]
Wow, awesome, and thanks for the quick turnaround on this one! 🚀 

It's really nice to see we're supporting that much honestly, always amazed.

Only a small nit comment, I think we could make the "code blocks" copy-able by setting a language to them? I suppose `sql` is a supported language.

File: content/en/user-guide/aws/rds/index.md [hunk 4]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
...
+CREATE EXTENSION IF NOT EXISTS aws_s3 CASCADE;
+SELECT aws_s3.query_export_to_s3(
+    'SELECT * FROM table2',
+    aws_commons.create_s3_uri('mybucket2', 'test.csv', 'us-east-1'),
+    options := 'FORMAT csv'
+)
+```
+
+### Additional extensions available in RDS PostgreSQL
```
Comment (HarshCasper) [inline content/en/user-guide/aws/rds/index.md:315]:
```suggestion
### Additional extensions
```

File: content/en/user-guide/aws/rds/index.md [hunk 5]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
...
+For example, in the SQL code snippet below, we are loading the `aws_lambda` extension, then generate a full ARN from a function name, and finally invoke the Lambda function directly from the SQL query:
+```
+CREATE EXTENSION IF NOT EXISTS aws_lambda CASCADE;
+-- create a Lambda function ARN
+SELECT aws_commons.create_lambda_function_arn('my_function');
+-- invoke a Lambda function directly from a SQL query
+SELECT aws_lambda.invoke('my_function', '{\"body\": \"Hello!\"}'::json);
+```
+
+### Using the `aws_s3` extension to interact with AWS S3
```
Comment (HarshCasper) [inline content/en/user-guide/aws/rds/index.md:291]:
```suggestion
### `aws_s3` extension
```

File: content/en/user-guide/aws/rds/index.md [hunk 6]
```diff
@@ -270,6 +270,59 @@ LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.
 - It's important to note that clusters removed from a global database lose their ability to function as standalone clusters, differing from their intended behavior on AWS.
 - At present, the capability for persistence within global databases is not available.
 
+## RDS PostgreSQL Extensions for AWS Service Integrations
+
+LocalStack supports certain extensions and functions that are provided in RDS to interact with other AWS services.
+At the moment, primarily extension functions for the PostgreSQL engine are supported.
+
+### Using the `aws_lambda` extension to interact with AWS Lambda
```
Comment (HarshCasper) [inline content/en/user-guide/aws/rds/index.md:278]:
```suggestion
### `aws_lambda` extension
```

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**